src/tests/ftest/daos_test/daos_core_test-rebuild.yaml

```diff
@@ -6,13 +6,25 @@ hosts:
     - server-B
     - server-C
     - server-D
-    - server-E
-    - server-F
-    - server-G
-    - server-H
 timeout: 600
 server_config:
+  servers_per_host: 2
   name: daos_server
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31416
+      log_file: daos_server0.log
+      scm_mount: /mnt/daos0
```

Comment:
Why not using SCM for rebuild tests?

Reply:
That work was intended to be addressed when working on DAOS-1457, but we could implement the change now.